example/spec/parallel/changeGranuleCollections/ChangeGranuleCollectionWorkflowSpec.js

```diff
@@ -35,11 +37,20 @@ const s3data = [
 const providersDir = './data/providers/s3/';
 const collectionsDir = './data/collections/s3_MOD09GQ_006_full_ingest';
 const targetCollectionsDir =
-  './data/collections/s3_MOD09GQ_007_full_ingest_move';
+  './data/collections/s3_MOD09GQ-AZ_006_full_ingest_move';
 
 const inputPayloadFilename =
   './spec/parallel/ingestGranule/IngestGranule.input.payload.json';
 
+async function getCMRClient(config) {
+  const lambdaFunction = `${config.stackName}-CreateReconciliationReport`;
```

Comment:
Nit: This .... is getting the CMR configuration directly from the deployed lambda environment variables.... which we are doing in other tests.       

We should probably, given how often we're doing this we consolidate it to a helper rather than boilerplate, but that doesn't have to be in this PR.